.gitignore

```diff
@@ -4,3 +4,4 @@ product-details-json
 thunderbird_notes
 thunderbird.net
 site
+.idea/*
```

Comment:
I'm not sure what this is for but it doesn't seem like something that should be committed.

Reply:
I use Jetbrains Phpstorm IDE. It uses .idea/* directories to store settings. So I added it to the .gitignore file. I figured it shouldn't hurt anything.

I used a separate commit so it could be dropped easily. So I'll make sure it isn't in the pull request when I update.